_pkgdown.yml

```diff
@@ -160,13 +140,44 @@ reference:
   - title: Utilities
     desc: Utility functions
     contents:
-      - match_output_arguments
       - run_region
       - expose_stan_fns
       - convert_to_logmean
       - convert_to_logsd
       - growth_to_R
       - R_to_growth
-      - add_day_of_week
       - update_secondary_args
+  - title: internal
```

Comment:
Do we need this section?

Reply:
Yes, otherwise pkgdown throws a warning (which is also how we know nothing is missing):
https://pkgdown.r-lib.org/reference/build_reference.html#missing-topics

Reply:
I think this reference talks about non-internal functions. However, see the next comment about using `has_keywords("internal")`.